### DISCUSSION TITLE
Audit & CLI Log Rotation Needed?

### DISCUSSION THREAD

**Discussion by dave-bell** [firefly-iii/firefly-iii#4469] [Firefly III] [answered]
I have Fire Fly running via docker and I just got caught out by the cron job that performs the automatic transfers.  I had set that up and it was running but as the wrong user.  As such files in `/var/www/html/storage/logs` weren't write-able by the web application.  I went and `chmod`ed the `cli` log file to the proper user and group things were all back to normal.

My question is about docker volume mounts.  At the moment I have provided a volume mount for `/storage/uploads` but I wonder about one for these logs as well?  Does the application itself perform log rotation on these?  I could certainly imagine the local drive running the actual container filling up.

Should I create a volume mount for `/storage/logs` and then setup a proper log retention policy?

**Comment by JC5** [comment, marked as answer]
Firefly III will automatically rotate these files.

**Reply by dave-bell** [reply]
Super cool!  Thanks.
